encord/orm/project.py

```diff
@@ -154,6 +154,11 @@ class ProjectCopyOptions(Enum):
     MODELS = "models"
 
 
+class ProjectWorkflowType(Enum):
```

Comment:
Great
This needs to be documented. 
I would prefer most documentation to be in code. You can see some other exampls with other enums. Then ideally you'd also find the right place in the docs where we talk about project creation and link to this workflow type.

Reply:
This one is internal implementation detail now, the configuration is done via the `ProjectWorkflowSettings` union type.